.gitignore

```diff
@@ -1,4 +1,42 @@
+### Bazel ###
```

Comment:
I think this commit belongs to #44 already?

Reply:
Sure, I cleaned up this branch. The change should be independent now.